kubernetes/base/rbac.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: descheduler-cluster-role
 rules:
-- apiGroups: [""]
+- apiGroups: ["events.k8s.io"]
```

Comment:
now that I am using client-go/tools/events, had to specify this apiGroup so our service account can create events